cloudformation/cfncluster.cfn.json

```diff
@@ -941,7 +941,7 @@
     "ScalingThreshold2" : {
       "Description" : "Threshold for triggering CloudWatch ScaleUp2 action",
       "Type" : "String",
-      "Default" : "200"
+      "Default" : "10"
```

Comment:
Please modify cloudformation/cfncluster.cfn.yaml file when modifying json file. we haven't automated this process. we were thinking of having only one format in github and the release process should generate another format while pushing templates to s3 bucket.